server/routes/admin/admin.js

```diff
@@ -46,24 +46,22 @@ module.exports = function(
 
     router.post('/roUsers/edit/:nomisId', async(async (req, res) => {
```

Comment:
If you're only awaiting inside a try/catch then you won't need the async middleware